README.md

```diff
@@ -222,6 +222,9 @@ See [docs/openai_api.md](docs/openai_api.md).
 ### Hugging Face Generation APIs
 See [fastchat/serve/huggingface_api.py](fastchat/serve/huggingface_api.py).
 
+## GPTQ 4bit Support
```

Comment:
Move this below this section?
https://github.com/lm-sys/FastChat#more-platforms

Reply:
Ok, thanks!